src/Lean/PrettyPrinter/Delaborator/Builtins.lean

```diff
@@ -606,7 +647,10 @@ partial def delabDoElems : DelabM (List Syntax) := do
       | Expr.lam _ _ body _ =>
         withBindingBodyUnusedName fun n => do
           if body.hasLooseBVars then
-            prependAndRec `(doElem|let $n:term ← $ma)
+            -- Note: the `$ma:term` annotation is required for elaboration to roundtrip
+            -- without indirecting through the parser. However, this annotation seems
+            -- to cause additional parentheses to be inserted in some cases.
```

Comment:
I only see *dropped* parentheses in the changed test output?

Reply:
Yes, good. I removed this comment.